docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,260 @@
+    # These args will get passed on to each operator
+    # You can override them on a per-task basis during operator initialization
+    default_args={
+        "depends_on_past": False,
+        "email": ["airflow@example.com"],
+        "email_on_failure": False,
+        "email_on_retry": False,
+        "retries": 1,
+        "retry_delay": timedelta(minutes=5),
+    },
+    description="A simple tutorial DAG",
+    schedule=timedelta(days=1),
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example"],
+) as dag:
+
+    # t1, t2 and t3 are examples of tasks created by instantiating operators
+    t1 = BashOperator(
+        task_id="print_date",
+        bash_command="date",
+    )
+
+    t2 = BashOperator(
+        task_id="sleep",
+        depends_on_past=False,
+        bash_command="sleep 5",
+        retries=3,
+    )
+    t1.doc_md = dedent(
+        """\
+    #### Task Documentation
+    You can document your task using the attributes `doc_md` (markdown),
+    `doc` (plain text), `doc_rst`, `doc_json`, `doc_yaml` which gets
+    rendered in the UI's Task Instance Details page.
+    ![img](http://montcs.bloomu.edu/~bobmon/Semesters/2012-01/491/import%20soul.png)
+    **Image Credit:** Randall Munroe, [XKCD](https://xkcd.com/license.html)
+    """
+    )
+
+    dag.doc_md = __doc__  # providing that you have a docstring at the beginning of the DAG; OR
+    dag.doc_md = """
+    This is a documentation placed anywhere
+    """  # otherwise, type it like this
+    templated_command = dedent(
+        """
+    {% for i in range(5) %}
+        echo "{{ ds }}"
+        echo "{{ macros.ds_add(ds, 7)}}"
+    {% endfor %}
+    """
+    )
+
+    t3 = BashOperator(
+        task_id="templated",
+        depends_on_past=False,
+        bash_command=templated_command,
+    )
+
+    t1 >> [t2, t3]
+```
+
+In order to rewrite this DAG in Dagster we will break it down into three parts
+
+1. The graph of compute
+2. The definition of the schedule
+3. The configuration of that graph and schedule
+
+### Defining the graph
+
+A [Job](/concepts/ops-jobs-graphs/jobs) is made up of a [Graph](/concepts/ops-jobs-graphs/graphs) of [Ops](/concepts/ops-jobs-graphs/ops). If you've used the Airflow taskflow api this will feel familiar, With ops you will be focused on writing a graph with python functions as the nodes with the data dependencies between those as edges.
+
+After rewriting the core compute of the tutorial Airflow Dag in Dagster we're left with:
+
+```python
+import time
+from datetime import date, timedelta
+
+from dagster import job, op
+
+@op
+def print_date():
+    dt = date.today()
+    print(dt)
+    return dt
+
+@op
+def sleep(dt: date):
+    time.sleep(5)
+
+@op
+def templated(dt: date):
+    for i in range(5):
+        print(dt)
+        print(dt - timedelta(days=7))
+
+@job
+def tutorial_job():
+    dt = print_date()
+    sleep(dt)
+    templated(dt)
+
+@repository
+def rewrite_repo():
+    return [tutorial_job]
```

Comment:
not mentioned in the airflow tutorial, we should not expose this yet. Maybe mention this in Step 4: Running locally?

I think this can be in a separate PR. Steps 4-7 is what's covered in the Testing section in the Airflow quickstart.